### DISCUSSION TITLE
Flying key feedback

### DISCUSSION THREAD

**Discussion by SteffeyDev** [danielbuechele/atemOSC#254] [General]
@reno- I looked at your flying key changes. You have all the necessary changes in FeedbackMonitors and Switcher.h, you just need to make some changes in Switcher.mm.  Find another monitor that is similar and follow how it is used in Switcher.mm.  You need to register the monitor in `switcherConnected` and de-register it in `cleanUpConnection`; that is what connects the monitor classes you've written to the switcher. If you continue copy-pasting down that path, you should find success.

**Comment by reno-** [comment]
I tried to implement feedback for pattern and fly in #255, but it does not compile. 
I have this error

<img width="1198" alt="Capture d’écran 2022-03-22 à 00 17 23" src="https://user-images.githubusercontent.com/127730/159379172-f922b031-f4f6-4bf6-9de9-703539e9644a.png">

I open a PR, hope that you might help me on this error

**Reply by reno-** [reply]
I think it works now, with this [commit](https://github.com/SteffeyDev/atemOSC/pull/255/commits/43aa95394dcc136c5e17fbae3b8e54adcd2f1b0f)

There is now another compilation error, that I don't really understand. Maybe someone could have a look ?
